src/app/experience/experience.component.css

```diff
@@ -4,7 +4,27 @@
 
 .about-me-text {
   font-size: 12pt;
-  color: darkgray;
+  color: black;
+}
+
+.experience-header {
+  font-size: 12pt;
+  font-style: oblique;
+  font-weight: normal;
+}
+
+.line {
+    border: none;
+    height: 1px;
+    /* Set the hr color */
+    color: #333; /* old IE */
+    background-color: darkgray; /* Modern Browsers */
+    width: 85%;
+}
+
+table.center {
+  margin-left:auto;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add one whitespace between the colon and the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=css%3Aformatting)